package.json

```
@@ -27,18 +27,18 @@
   "homepage": "https://github.com/digitalbazaar/bedrock-mail",
   "dependencies": {
     "app-root-path": "^3.0.0",
-    "ejs": "^3.0.1",
-    "email-templates": "^7.0.4",
-    "nodemailer": "^6.4.3"
+    "ejs": "^3.1.6",
+    "email-templates": "^8.0.4",
+    "nodemailer": "^6.6.0"
   },
   "peerDependencies": {
-    "bedrock": "1.6.0 - 3.x"
+    "bedrock": "4.x"
```

Use the "^" syntax when not using ranges.  I'm guessing 4.0.0 is ok, but adjust upward if needed. 
```suggestion
    "bedrock": "^4.0.0"
```

---

Done, thanks.